docs/schema-v2.md

```diff
@@ -126,9 +126,17 @@ Only the `square-textured` skybox type uses these fields
 |------------|-----------------|---------------------------------------------------------------------------------------------------------|--------------------|---------------|
 | `textures` | Textures object | Specifies the textures to be used for each cardinal direction.                                          | :white_check_mark: |       -       |
 
+### Extra fields used by `animate-square-textured`
+Only the `animate-square-textured` skybox type uses these fields
+
+| Name       | Json Datatype   | Description                                                                                             | Required           | Default value |
+|------------|-----------------|---------------------------------------------------------------------------------------------------------|--------------------|---------------|
```

Comment:
Please make the list line up here properly if you can :P

Reply:
it actually doesn't matter. it'll show up properly. 
I used https://www.tablesgenerator.com/markdown_tables to create the clean-looking tables